app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -18,19 +18,18 @@ import {
 import DesktopAuthBg from "features/auth/resources/desktop-auth-bg.jpg";
 import MobileAuthBg from "features/auth/resources/mobile-auth-bg.jpg";
 import useAuthStyles from "features/auth/useAuthStyles";
-import React from "react";
 import { Link } from "react-router-dom";
 import { loginRoute, signupRoute } from "routes";
 
 import { COUCHERS } from "../../constants";
 
 const useStyles = makeStyles((theme) => ({
   aboutUs: {
-    color: "#2a2a2a",
-    marginTop: "auto",
+    color: theme.palette.common.white,
+    marginTop: theme.spacing(2),
```

Comment:
Ok plus a cheeky fix - the "about us" link on mobile screens were too close to the two buttons before this fix.